test/cri-containerd/syscall.go

```diff
@@ -0,0 +1,7 @@
+// +build functional
```

Comment:
Do we want this to only be in the test dir? Might be useful to move it out so we can create something like the "create-scratch" utility for ntfs vhds in the future

Reply:
Fixed.